types/rest/rest.go

```diff
@@ -24,6 +24,13 @@ const (
 	DefaultLimit = 30 // should be consistent with tendermint/tendermint/rpc/core/pipe.go:19
 )
 
+// ResponseWithHeight defines a response object type that wraps an original
+// response with a height.
+type ResponseWithHeight struct {
```

Comment:
Please add a constructor for this type

Reply:
We don't really need one -- idk where the notion of require a `New*` for every single type came from...